includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -15,8 +16,28 @@ public static function getInstance() {
 		return static::$instance;
 	}
 
+	private static function mapHrefToLocalUrl($href) {
+		$href = WikiFactory::getLocalEnvURL($href);
+
+		// We need to change 'https' to 'http' for stable/preview/verify/sandbox environments cause
+		// we do not have ssl certificate for these subdomains
+		$regex = '/^https:\/\/(stable|preview|verify|sandbox-[a-z0-9]+)\.[^\/]+\.wikia\.com/';
```

Comment:
what do you think about checking `$wgWikiaEnvironment` value instead of regexing on URLs?